src/BQ24195.cpp

```diff
@@ -1195,6 +1202,68 @@ bool PMICClass::isBatteryInOverVoltage() {
     return 0;
 }
 
+
+/*******************************************************************************
+ * Function Name  : setFastChargeTimerSetting
+ * Description    : Sets the charging safety timer
+ * Input          : safety timer duration in hours (rounds up)
+ * Return         : 0 on Error, 1 on Success
+ *******************************************************************************/
+bool PMICClass::setFastChargeTimerSetting(float hours) {
```

Comment:
```suggestion
bool PMICClass::setFastChargeTimerSetting(float const hours) {
```
Try and use const-correctness as much as possible :wink: